### ISSUE TITLE
Open in Tabs: Update blog view selector

### ISSUE THREAD

**Issue by marcustyphoon** [AprilSylph/XKit-Rewritten#751]
### Concept

> Blog view URLs have been changed. Instead of /blog/view/blogname they’re now simply /blogname (e.g. tumblr.com/changes).

This will probably necessitate a change to this line in Open in Tabs:

https://github.com/AprilSylph/XKit-Rewritten/blob/cee9850ca027248c86f6298bff18717a32777a6c/src/scripts/open_in_tabs.js#L15

I don't know if this currently does anything.

**Comment by AprilSylph** [comment]
I actually helped implement this new route format and it does suck for this. There's no way to tell from the outside if a link is to a blog or not just by the `href`. Redpop has a list of reserved names but it's not exposed and the long-term goal is to remove it anyway.

**Comment by marcustyphoon** [comment]
Yeah, I was thinking of maybe just regexing `tumblr.com/anyString/obviouslyAPostId` and dropping support for direct links to blogs. Not familiar offhand with every place this code actually matters.

**Comment by AprilSylph** [comment]
Now hold on, I just realised because blog links are generally standardised, we could target `[data-login-wall-type="blogView"]` for most cases. Just checked and they are present on notifications.

Also, the regex for blog names off the top of my head is `/[a-z0-9-]{1,32}/`, if we do need it.
